packages/models-library/src/models_library/api_schemas_webserver/users.py

```diff
@@ -0,0 +1,130 @@
+import re
+from datetime import date
+from enum import Enum
+from typing import Annotated, Literal
+
+from models_library.api_schemas_webserver.groups import MyGroupsGet
+from models_library.api_schemas_webserver.users_preferences import AggregatedPreferences
+from models_library.basic_types import IDStr
+from models_library.emails import LowerCaseEmailStr
+from models_library.users import FirstNameStr, LastNameStr, UserID
+from pydantic import BaseModel, ConfigDict, Field, field_validator
+
+from ._base import InputSchema, OutputSchema
+
+
+class ProfilePrivacyGet(OutputSchema):
+    hide_fullname: bool
+    hide_email: bool
+
+
+class ProfilePrivacyUpdate(InputSchema):
```

Comment:
this is a Patch or an Update? if that is the usual naming scheme, then forget that comment

Reply:
Update = PATCH, Replace = PUT.
In any case, I notice we get confused between update and replace. I guess it means we should perpahs start using Patch.

So the model names should be : `$ResourceName$Operation` where `Operation in [Get, Update, Replace, Delete, Create]`.   Perhaps instead of `Update` we should use `Patch`?

Reply:
Will follow up in https://github.com/ITISFoundation/osparc-simcore/issues/6913